gradle/libs.versions.toml

```
@@ -0,0 +1,41 @@
+[versions]
+agp = "7.1.2"
+bytebuddy = "1.12.10"
+jcodec = "0.2.5"
+kotlin = "1.6.10"
+moshi = "1.13.0"
+
+[libraries]
+# Gradle plugins
+androidGradlePlugin = { module = "com.android.tools.build:gradle", version.ref = "agp" }
+dokkaGradlePlugin = { module = "org.jetbrains.dokka:dokka-gradle-plugin", version = "1.7.10" }
+grgitGradlePlugin = { module = "org.ajoberstar.grgit:grgit-gradle", version= "5.0.0" }
+kotlinGradlePlugin = { module = "org.jetbrains.kotlin:kotlin-gradle-plugin", version.ref = "kotlin" }
+mavenPublishGradlePlugin = { module = "com.vanniktech:gradle-maven-publish-plugin", version = "0.21.0" }
+spotlessGradlePlugin = { module = "com.diffplug.spotless:spotless-plugin-gradle", version = "6.8.0" }
```

```suggestion
plugins-spotless = { module = "com.diffplug.spotless:spotless-plugin-gradle", version = "6.8.0" }
```

and similar, i.e., no need to reference Gradle, since that's the build tool that consumes this toml and move `Plugins` suffix to `plugins-` prefix.

---

`plugins` is a reserved prefix because of `[plugins]` 😅, but `plugin` works, and makes even more sense.
```
  - Problem: In version catalog libs, alias 'plugins-grgit' is not a valid alias.

    Reason: Prefix for dependency shouldn't be equal to 'plugins'.

    Possible solution: Use a different alias which prefix is not equal to 'bundles', 'plugins' or 'versions'.

    Please refer to https://docs.gradle.org/7.5.1/userguide/version_catalog_problems.html#reserved_alias_name for more details about this problem.
```

fixed in https://github.com/cashapp/paparazzi/pull/571/commits/80bca8d9068dca5ae7c7ce7323431a01a467eace on #571